lib/util/lib/platform.c

```diff
@@ -190,9 +189,9 @@ hse_cpu_init(void)
 
     hse_tsc_mult = (NSEC_PER_SEC << HSE_TSC_SHIFT) / hse_tsc_freq;
 
-    log_info("bogomips %d, freq %lu, shift %u, mult %u, L1D_CLSZ %d",
-             bogomips, hse_tsc_freq, HSE_TSC_SHIFT, hse_tsc_mult,
-             LEVEL1_DCACHE_LINESIZE);
+    log_info(
+        "bogomips %d, freq %lu, shift %u, mult %u, L1D_CLSZ %d", bogomips, hse_tsc_freq,
+        HSE_TSC_SHIFT, hse_tsc_mult, LEVEL1_DCACHE_LINESIZE);
```

Comment:
train wreck...

Reply:
Yeah.  Not sure how that didn't get called out on earlier rounds.  Fixed with this:
```
-AlignAfterOpenBracket: AlwaysBreak
+AlignAfterOpenBracket: DontAlign
```